torch_xla/distributed/xla_multiprocessing.py

```diff
@@ -215,7 +215,7 @@ def _pre_fork_setup(num_devices):
   return PreForkConfig(dev_kind=dev_kind, num_devices=num_devices)
 
 
-def _setup_gpu_worker(index, gindex, pf_cfg):
```

Comment:
IMHO when there are symmetrical APIs (gpu, tpu, cpu) it is better to have a consistent interface, even though one instance of such APIs does not actually use an argument.

Reply:
I'm not sure what consistency you're referring to here. All gpu, tpu, cpu consistently don't use that argument.

Reply:
Right. That argument contains the configuration info parsed before the fork.
Currently it is not used by the `_setup_*_worker` APIs.
IMHO it is better to pass that down, but if you want to remove the argument that is not a big deal for me.

Reply:
I'll keep it removed if that's fine by you. Thanks for the review Davide!